src/test/java/elizCurtnazar/tests/tema6/DemoQaWindow.java

```diff
@@ -0,0 +1,139 @@
+package elizCurtnazar.tests.tema6;
+
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.ElementClickInterceptedException;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import teofilursan.driver.BrowserManager;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Set;
+
+public class DemoQaWindow {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) throws IOException {
+        navigateToDemoQAWindowPage();
+//        clickOnNewTabButton(); EXEMPLU DIN ORE
+        clickNewTab();
+        clickNewWindow();
+//        clickNewWindowMessage();
+        try {
+            clickNewWindowMessage();
+            System.out.println("Am intrat in blocul de try.");
+        } catch (Exception | Error e) {
+            System.out.println("Am prins o exceptie in blocul de catch.");
+            File file = driver.getScreenshotAs(OutputType.FILE);
```

Comment:
Use FileUtils.takeScreenshot method instead of these lines to avoid code duplication. Thanks!

Reply:
done